tests/containers/install_updates.pm

```diff
@@ -13,14 +13,37 @@ use utils;
 use power_action_utils;
 use version_utils qw(check_os_release get_os_release is_sle);
 
+sub sanity_chk_install {
```

Comment:
Let's use a full name here considering it's a library name, not a temporary variable, and 2 characters are not worth saving.

```suggestion
sub sanity_check_install {
```